src/components/Books.js

```diff
@@ -1,19 +1,24 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Please do not disable linters, they are there to help you write better code. Do not do that unless you have a strong reason which prevents you from achieving the result